common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -216,7 +216,8 @@ def get_html(self):
         # Determine if there is an alternative source for this video
         # based on user locale.  This exists to support cases where
         # we leverage a geography specific CDN, like China.
-        cdn_url = getattr(settings, 'VIDEO_CDN_URL', {}).get(self.system.user_location)
+        default_cdn_url = getattr(settings, 'VIDEO_CDN_URL', {}).get('default')
+        cdn_url = getattr(settings, 'VIDEO_CDN_URL', default_cdn_url).get(self.system.user_location)
```

Comment:
This won't quite work.  The value of `default_cdn_url` is a string.  But the value returned by the `getattr(settings, 'VIDEO_CDN_URL',...` should be a dictionary.

I think what you want is:
```
cdn_url = getattr(settings, 'VIDEO_CDN_URL', {}).get(self.system.user_location, default_cdn_url)
```

Reply:
I don't understand why default_cdn_url is a string but cdn_url is a dictionary.   Shouldn't they both be dictionaries?

Reply:
oh, nevermind, I had the getattr and get actions backwards.